src/extensions/Mention.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<NodeViewWrapper as="span" class="mention" contenteditable="false">
-		<NcUserBubble :user="node.attrs.id" :display-name="username" class="mention-user-bubble">
+		<NcUserBubble :user="node.attrs.id" :display-name="username" :primary="isCurrentUser()" class="mention-user-bubble">
```

Comment:
Could you try to avoid call functions inside templates?
Maybe computed works, `this.node` is reactive.